modules/products/static-site/static-site/cloudfront.tf

```diff
@@ -0,0 +1,52 @@
+resource "aws_cloudfront_origin_access_identity" "static_site_identity" {
```

Comment:
Do we want to move to ws_cloudfront_origin_access_control as this has superseded OAI from AWS recommendations